### PR TITLE
F upgrade storybook migrate dp button row

### PR DESCRIPTION
**Pull request by ahmad-demos**
https://github.com/demos-europe/demosplan-ui/pull/379

- Migrate DpButtonRow stories to v7.
- Add de Trans-Key.

Screenshot 2023-07-31 at 13-11-23 Components _ ButtonRow - Default ⋅ Storybook

### CODE REVIEW

File: src/components/shared/translations.js [hunk 1]
```diff
@@ -5,7 +5,9 @@ const de = {
   errorMandatoryFieldsIntro: 'Bitte korrigieren Sie Ihre Eingabe in folgenden Feldern: ',
   errorMandatoryFieldsOutro: '. Erst dann können Sie Ihre Angaben speichern.',
   errorMandatoryFieldsDefault: 'Bitte füllen Sie alle Pflichtfelder korrekt aus. Erst dann können Sie Ihre Angaben speichern.',
-  placeholderAutoSuggest: "Suchbegriff..."
+  placeholderAutoSuggest: "Suchbegriff...",
+  save: "Speichern",
+  abort: "Abbrechen"
```
Comment (spiess-demos) [inline src/components/shared/translations.js:10]: Please let's use the term "cancel" as key here. While the german term "Abbrechen" is the correct translation for both "Abort" and "Cancel", the term "Abort" is much more harsh, indicating the immediate termination of a process that has already begun, while "Cancel" mildly implies to return to the previous state, which i feel is more default here.

Also, i do not like the term "abort" for its other connotations, be them in english or german.

**Review by hwiem — COMMENTED** [review]
I'm getting errors when running Storybook on this branch, or is that a known issue?

**Comment by ahmad-demos** [comment]
> I'm getting errors when running Storybook on this branch, or is that a known issue?

Which error do you have? I don't have any error!

**Comment by hwiem** [comment]
> > I'm getting errors when running Storybook on this branch, or is that a known issue?
> 
> Which error do you have? I don't have any error!

I get this error:
`Cannot find module '../../dplan/projects/diplanbau/web/css/style.0f5e11.css'`

**Comment by ahmad-demos** [comment]
> I'm getting errors when running Storybook on this branch, or is that a known issue?

Which error do you have? I don't have any error!

> > > I'm getting errors when running Storybook on this branch, or is that a known issue?
> > 
> > 
> > Which error do you have? I don't have any error!
> 
> I get this error: `Cannot find module '../../dplan/projects/diplanbau/web/css/style.0f5e11.css'`

Yes this is ok, see in https://github.com/demos-europe/demosplan-ui/pull/379#discussion_r1267882434 please!
To test yo can copy another css file from your local project folder in `web/css`

**Comment by hwiem** [comment]
> > I'm getting errors when running Storybook on this branch, or is that a known issue?
> 
> Which error do you have? I don't have any error!
> 
> > > > I'm getting errors when running Storybook on this branch, or is that a known issue?
> > > 
> > > 
> > > Which error do you have? I don't have any error!
> > 
> > 
> > I get this error: `Cannot find module '../../dplan/projects/diplanbau/web/css/style.0f5e11.css'`
> 
> Yes this is ok, see in [#379 (comment)](https://github.com/demos-europe/demosplan-ui/pull/379#discussion_r1267882434) please! To test yo can copy another css file from your local project folder in `web/css`

I think that kind of information should be provided in the PR description in the "How to test" section

**Review by hwiem — APPROVED** [review]
lgtm, code review only

**Pull request merged**